scripts/ci/upload_artifact.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+import os
+import paramiko
+import sys
+from io import StringIO
+
+def upload_files(remote_host, remote_user, remote_port, ssh_private_key, source, target):
+    pk = StringIO(ssh_private_key)
+    ssh_key = paramiko.Ed25519Key.from_private_key(pk)
```

Comment:
nit: could we use `sk` here. I always associate `pk` with pubkey but maybe that's only me.